python/lsst/sims/GalSimInterface/galSimPSF.py

```diff
@@ -199,11 +199,11 @@ def __init__(self, airmass=1.2, rawSeeing=0.7, band='r'):
         FWHMsys = numpy.sqrt(0.25**2 + 0.3**2 + 0.08**2) * airmass ** 0.6
         # From LSST-20160 eqn (4.2)
 
-        atm = galsim.Kolmogorov(fwhm=FWHMatm)
-        sys = galsim.Gaussian(fwhm=FWHMsys)
+        atm = galsim.Kolmogorov(fwhm=FWHMatm, gsparams=gsparams)
+        sys = galsim.Gaussian(fwhm=FWHMsys, gsparams=gsparams)
```

Comment:
Probably even better than Kolmogorov * Gaussian would be Komogorov * Airy.  Josh's atmospheric PSF already uses the Airy component as an approximation for a few calculations related to the aperture.  So if you used the same lam, diam, and obscuration as for the full atmospheric PSF, then I think Kolmogorov * Airy would be *very* close to the right size.

OTOH, this may already be accurate enough, so maybe not worth adding another version.